### DISCUSSION TITLE
TLS 客户端验证的可能性

### DISCUSSION THREAD

**Discussion by 0xffffharry** [v2fly/v2ray-core#1808] [Ideas]
开发者是否有可能加上TLS客户端验证？如果加上TLS双向验证，连接安全性也有保证，且在连接内也可以使用更轻便的协议，比如vless socks

**Comment by ioctltty** [comment]
我也有这个想法。我看到代码里已经有[客户端证书的选项](https://github.com/v2fly/v2ray-core/blob/master/transport/internet/tls/config.proto#L62)了，不过现有的文档里貌似还没有提及具体用法。

VLESS貌似被抛弃了。内层用什么协议其实无所谓，能做流量转发就行。
